Add migration/migrator_spec

donaldong opened the pull request:
This mocks a rails application initialization process. It loads the config and runs `rake redis:migrate`in a spec environment.

katyho commented on spec/migration/migrator_spec.rb:
```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
```
Is it valuable to also test that the migration version stored in redis is changed?

donaldong replied:
yeah good idea!

donaldong commented:
expect the migration versions on redis have changed

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/70e3ba044b01e7cab6cd8a729366762d5e960758..1e8e399e5eec33a38d8bb5d17d674d768be1bf4c).
